docs/getting-started.md

```diff
@@ -0,0 +1,96 @@
+Getting Started
+===============
+
+Repository Requirements
+-----------------------
+
+VFS for Git will work with any Git service that supports the
+[GVFS protocol](/Protocol.md). For example, you can create a repo in
+[Azure DevOps](https://azure.microsoft.com/services/devops/), and push
+some contents to it. There are two constraints:
+
+  * Your repo must not enable any clean/smudge filters
+  * Your repo must have a `.gitattributes` file in the root that includes
+    the line `* -text`
+
+
+Cloning 
+-------
+
+The `clone` verb creates a local enlistment of a remote repository using the
+[GVFS protocol](https://github.com/microsoft/VFSForGit/blob/master/Protocol.md).
+
+```
+gvfs clone [options] <url> [<dir>]
+```
+
+Create a local copy of the repository at `<url>`. If specified, create the `<dir>`
+directory and place the repository there. Otherwise, the last section of the `<url>`
+will be used for `<dir>`. At the end, the repo is located at `<dir>/src`.
+
+### Options
+
+These options allow a user to customize their initial enlistment.
+
+* `--cache-server-url=<url>`: If specified, set the intended cache server to
+  the specified `<url>`. All object queries will use the GVFS protocol to this
+  `<url>` instead of the origin remote. If the remote supplies a list of
+  cache servers via the `<url>/gvfs/config` endpoint, then the `clone` command
+  will select a nearby cache server from that list.
+
+* `--branch=<ref>`: Specify the branch to checkout after clone.
+
+* `--local-cache-path=<path>`: Use this option to override the path for the
+  local Scalar cache. If not specified, then a default path inside
+  `<Volume>:\.gvfsCache\` is used. The default cache path is recommended so
+  multiple enlistments of the same remote repository share objects on the

```

Comment:
```suggestion
  multiple clones of the same remote repository share objects on the

```

Reply:
I change this here, but elsewhere we use the word "enlistment" on purpose (and it matches internal concepts that are distinct from worktrees or clones).